projects/plugins/wpcomsh/Makefile

```
@@ -102,8 +102,8 @@ build: check clean $(BUILD_DST)/$(BUILD_FILE)
 
 ## i18n
 i18n:
-	@ /bin/bash $(BUILD_SRC)/bin/i18n/generate-pot.sh
-	@ /bin/bash $(BUILD_SRC)/bin/i18n/update-translations.sh
+	@ /usr/bin/env bash $(BUILD_SRC)/bin/i18n/generate-pot.sh
+	@ /usr/bin/env bash $(BUILD_SRC)/bin/i18n/update-translations.sh
```

There's definitely no reason to use `/usr/bin/env` here. Just doing `bash` will already search the path.
```suggestion
	@ bash $(BUILD_SRC)/bin/i18n/generate-pot.sh
	@ bash $(BUILD_SRC)/bin/i18n/update-translations.sh
```